src/download.jl

```diff
@@ -121,6 +123,11 @@ function download(url::AbstractString, local_path=nothing, headers=Header[]; upd
         start_time = now()
         prev_time = now()
 
+        if header(resp, "Content-Encoding") == "gzip"
+            stream = TranscodingStream(GzipDecompressor(), stream) # auto decoding
```

Comment:
I think this can just be:
```suggestion
            stream = GzipDecompressorStream(stream) # auto decoding
```
and then we don't need the explicit dependency on TranscodingStreams

Reply:
Thank you. I fixed.